mysql/bin/wait_for_mysql

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# The Jenkins runner doesn't support TTY sessions meaning docker-compose exec commands
+# failed to execute properly and would hang the CI
+# https://github.com/docker/compose/issues/5696
+until docker-compose exec -T db mysql -hdb -uroot -proot -e 'SELECT 1' &> /dev/null
+do
+  printf "."
+  sleep 1
+done
```

Comment:
Has this been working for you? I remember we had problems getting this to be consistent.

Reply:
Yup this script works fine for this